lib/core-integration/src/Test/Integration/Scenario/API/ByronWallets.hs

```diff
@@ -188,7 +189,53 @@ spec = do
                     getFromResponse amount r0
             actualFee `shouldBe` predictedFee
 
-    describe "BYRON_ESTIMATE_06 - non-existing wallets" $  do
+    it "BYRON_MIGRATE_04 - \
+        \a migration operation removes all funds from the source wallet."
+        $ \ctx -> do
+
+            -- Restore a Byron wallet with funds, to act as a source wallet:
+            let sourceWalletName = "source wallet"
+            let sourceWalletPass = "source wallet passphrase"
+            sourceWallet <-
+                fixtureByronWalletWith sourceWalletName sourceWalletPass ctx
+
+            -- Verify that the source wallet has funds available:
+            r0 <- request @ApiByronWallet ctx
+                (getByronWalletEp sourceWallet) Default Empty
+            verify r0
+                [ expectResponseCode @IO HTTP.status200
+                , expectFieldSatisfy balanceAvailable (> 0)
+                ]
```

Comment:
This check is superfluous, that is what `fixture*Wallet*` gives you already. The function only returns when funds are available.

Reply:
> This check is superfluous, that is what `fixture*Wallet*` gives you already. The function only returns when funds are available.

Fair enough. I added this as a sanity check, but we can remove it if we're confident that the `fixtureByronWallet` will only ever return a wallet with non-zero funds.

Reply:
We are :)

Reply:
Removed!